pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -670,9 +670,11 @@ mod minting {
 		.unwrap();
 		assert!(avatar_ids.is_full());
 
+		let rarity_tiers = test_rarity_tiers(vec![(RarityTier::Common, 100)]);
+
 		ExtBuilder::default()
 			.organizer(ALICE)
-			.seasons(vec![Season::default()])
+			.seasons(vec![Season::default().rarity_tiers(rarity_tiers)])
```

Comment:
(nitpick) why don't build seasons instead (for legibility)? 
`		let seasons = vec![
			Season::default().rarity_tiers(test_rarity_tiers(vec![(RarityTier::Common, 100)]))
		];

		ExtBuilder::default()
			.organizer(ALICE)
			.seasons(seasons)`

Reply:
Also, I think I would create a different test for both conditions: max num of avatars and max rarity reached.